pkg/config/config.go

```diff
@@ -702,10 +702,10 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("logs_config.use_http", false)
 	config.BindEnvAndSetDefault("logs_config.use_tcp", false)
 
-	bindEnvAndSetLogsConfigKeys(config, "logs_config.", false)
-	bindEnvAndSetLogsConfigKeys(config, "database_monitoring.samples.", false)
-	bindEnvAndSetLogsConfigKeys(config, "database_monitoring.metrics.", false)
-	bindEnvAndSetLogsConfigKeys(config, "network_devices.metadata.", false)
+	bindEnvAndSetLogsConfigKeys(config, "logs_config.")
+	bindEnvAndSetLogsConfigKeys(config, "database_monitoring.samples.")
+	bindEnvAndSetLogsConfigKeys(config, "database_monitoring.metrics.")
+	bindEnvAndSetLogsConfigKeys(config, "network_devices.metadata.")
```

Comment:
Does it mean all [EP Forwarder pipelines](https://github.com/DataDog/datadog-agent/blob/main/pkg/epforwarder/epforwarder.go#L30-L57) will all use v2 by default ?

For now, network-devices-metadata pipeline is not ready to use v2 format, due to our custom network-devices-intake (similar to DBM metrics intake) compatible with EP forwarder v1 only.

Reply:
No, this should not affect network devices. To use v2 intake a track type must also be defined: see [here](https://github.com/DataDog/datadog-agent/blob/9dbf1894b607bb13e59b037395c8b4771799698b/pkg/logs/config/config.go#L215) and [here](https://github.com/DataDog/datadog-agent/blob/9dbf1894b607bb13e59b037395c8b4771799698b/pkg/epforwarder/epforwarder.go#L44)

Reply:
Thanks, missed that. Looks fine then.

(minor) It can possibly help to have a comment for [passthroughPipelineDesc. intakeTrackType](https://github.com/DataDog/datadog-agent/blob/9dbf1894b607bb13e59b037395c8b4771799698b/pkg/epforwarder/epforwarder.go#L139) is a v2 config and will trigger the use of v2 API.